extensions/wikia/AdEngine/js/lookup/openXBidder.js

```diff
@@ -5,10 +5,11 @@ define('ext.wikia.adEngine.lookup.openXBidder', [
 	'ext.wikia.adEngine.lookup.lookupFactory',
 	'ext.wikia.adEngine.slot.adSlot',
 	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'ext.wikia.adEngine.lookup.openx.openXBidderHelper',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (adContext, factory, adSlot, adLogicZoneParams, doc, log, win) {
+], function (adContext, factory, adSlot, adLogicZoneParams, openXHelper, doc, log, win) {
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.lookup.openXBidder',
```

Comment:
What do you think about moving config with slot map and a few method from here to new openx module and manage slot list only from that module?